src/apps/api/urls.py

```diff
@@ -16,6 +16,8 @@
 router.register('datasets', datasets.DataViewSet)
 router.register('data_groups', datasets.DataGroupViewSet)
 router.register('leaderboards', leaderboards.LeaderboardViewSet)
+router.register('tasks', tasks.TaskViewSet)
+router.register('tasksearch', tasks.TaskViewSetSimple)
```

Comment:
What's the difference between `TaskViewSet` vs `TaskViewSetSimple`?

Reply:
I know I created `Simple` for working with the Phase Modal, trying to get Select2 to work. Since we kinda jumped ship on that one and put me to work on v1.5 -> v2 stuff (i.e. things I understood how to do), I don't think this ever got optimized (or maybe even working), so I'm not super sure `Simple` is necessary at all.

Reply:
cool, gotchya, could you confirm we don't need it any more and pull it or at least comment it out?

Reply:
So this was created for Select2 to dynamically populate its dropdown via searching. It needs a serializer that has specific fields. Could _maybe_ be done with a get serializer context? but I don't know how we would differentiate which serializer gets called.

Reply:
Hmm, do you think you could make a custom view with `@action` or switch the serializer in `get_serializer_class`? Maybe this is an OK way for now.. hmm...

Reply:
Possibly, I'm not super sure what the logic of it all would be, especially if we want to search things different on a different screen? like task management screen. I can tag the serializers and views with a TODO so we think about it when we jump back in to this portion of things?

Reply:
👍